elements/pfe-accordion/BaseAccordionPanel.ts

```diff
@@ -0,0 +1,47 @@
+import { LitElement, html } from 'lit';
+import { property } from 'lit/decorators.js';
+
+import { colorContextConsumer } from '@patternfly/pfe-core/decorators.js';
+import { getRandomId } from '@patternfly/pfe-core/functions/random.js';
+
+import { PfeCollapsePanel } from '@patternfly/pfe-collapse/pfe-collapse-panel.js';
+
+import style from './BaseAccordionPanel.scss';
+import type { ColorTheme } from '@patternfly/pfe-core';
+
+export class BaseAccordionPanel extends LitElement {
+  static readonly styles = [
+    ...PfeCollapsePanel.styles,
```

Comment:
we're going to remove collapse, so maybe now's the best time to move those styles over. We can defer this until after but doing the work here will make it a drop easier.